### DISCUSSION TITLE
Error at install on fresh Debian 10 VPS

### DISCUSSION THREAD

**Discussion by jbenguira** [coder/code-server#2373] [General]
When I try to install code-server on a fresh Debian 10 VPS I get an error

root@d:/# curl -fsSL https://code-server.dev/install.sh | sh

Debian GNU/Linux 10 (buster)
Installing v3.7.3 deb package from GitHub releases.

+ mkdir -p ~/.cache/code-server
+ curl -#fL -o ~/.cache/code-server/code-server_3.7.3_amd64.deb.incomplete -C - https://github.com/cdr/code-server/releases/download/v3.7.3/code-server_3.7.3_amd64.deb
######################################################################## 100.0%
curl: (22) The requested URL returned error: 416

Any idea about what's wrong and how to fix it?

Thanks

**Comment by nhooyr** [comment]
Install's fine for me, I think something's wrong with your network.

Try creating a new VPS or one on a different region.

**Reply by sr229** [reply]
Can confirm this is a network issue. This is not a issue we can fix.
